src/cli/api/apihelper.go

```diff
@@ -0,0 +1,137 @@
+package api
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"autoscaler/models"
+	e "cli/errors"
+	"cli/ui"
+)
+
+const (
+	HealthPath  = "/health"
+	PolicyPath  = "/v1/apps/{appId}/policy"
+	MetricPath  = "/v1/apps/{appId}/metric_histories/{metric_type}"
+	HistoryPath = "/v1/apps/{appId}/scaling_histories"
+)
+
+type APIHelper struct {
+	Endpoint *APIEndpoint
+	Client   *CFClient
+}
+
+func (helper *APIHelper) DoRequest(req *http.Request, host string) (*http.Response, error) {
+
+	client := &http.Client{
+		Timeout: 30 * time.Second,
+		Transport: &http.Transport{
+			DisableKeepAlives: true,
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: helper.Endpoint.SkipSSLValidation,
+			},
+			Proxy:               http.ProxyFromEnvironment,
```

Comment:
do you mean to use default client instead of declaring it explicitly? 
but there is no InsecureSkipVerify setting in DefaultTransport, then we can't define skip-ssl-validation ...